lib/web/jquery/jquery-migrate.js

```diff
@@ -260,7 +260,7 @@ jQuery.fn.init.prototype = jQuery.fn;
 jQuery.find = function( selector ) {
 	var args = Array.prototype.slice.call( arguments );
 
-	// Support: PhantomJS 1.x
+	// SupportMessage: PhantomJS 1.x
```

Comment:
@holmatro90 The same as above. There files must not be changed without the real need  and must not be committed due to the accidental changes.